src/main/java/mate/academy/hibernate/relations/model/Country.java

```diff
@@ -1,6 +1,16 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "country")
 public class Country implements Cloneable {
```

Comment:
Implementing Cloneable and overriding `clone` method is not typical for entity classes. Entities should be unique and managed by Hibernate, which means manual cloning could lead to unexpected behavior, especially with regards to identity and state management. If you need to copy properties, consider using copy constructors or factory methods.